docs/conf/australia/2022/welcome-wagon.rst

```diff
@@ -0,0 +1,261 @@
+:template: {{year}}/generic.html
+
+Welcome Wagon Guide
```

Comment:
Not sure why this is hardcoded in the website, we extracted a lot of shared content due to the virtual platform being identical. You can look at the Prague website directory and copy as many of the include statements (Writing Day, Unconf, Lightning Talks, WW, etc).